src/components/GradeLevelFilter.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Popover } from '@material-ui/core';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    height: theme.spacing(4),
+  },
+  popOverWindow: {
+    minWidth: '200px',
+  },
+}));
+
+const gradeList = {
+  12: {
+    value: 'Highschool Senior',
+    isChecked: false,
+  },
+  13: {
+    value: 'College Freshman',
+    isChecked: false,
+  },
+  14: {
+    value: 'College Sophomore',
+    isChecked: false,
+  },
+  15: {
+    value: 'College Junior',
+    isChecked: false,
+  },
+  16: {
+    value: 'College Senior',
+    isChecked: false,
+  },
+};
+
+export default function GradeLevelFilter() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+
+  return (
+    <>
+      <Button
+        variant="outlined"
+        className={classes.buttonStyle}
+        onClick={handleClick}
+        endIcon={<ArrowDropDownIcon color="primary" />}>
+        Grade
+      </Button>
+      <Popover
+        className={classes.popOverWindow}
+        open={Boolean(anchorEl)}
+        anchorEl={anchorEl}
+        onClose={() => setAnchorEl(null)}
+        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+        transformOrigin={{ vertical: 'top', horizontal: 'left' }}>
+        <input
+          key="12"
+          type="checkbox"
+          // eslint-disable-next-line no-return-assign
+          onClick={() => (gradeList[12].isChecked = !gradeList[12].isChecked)}
+          checked={gradeList[12].isChecked}
+          value={gradeList[12].value}
+        />
+        {gradeList[12].value}
```

Comment:
Im thinking this would be iterated through the gradeList object and then for each then there would be an input applied, is what im thinking at the moment to get the checkbox to be rendered for all the grades in the gradeList